public/locale/en.json

```
@@ -2149,6 +2151,7 @@
   "outgoing": "Outgoing",
   "overview": "Overview",
   "oxygen_information": "Oxygen Information",
+  "oxygen_saturation": "SpO2",
```

```suggestion
  "spo2": "SpO2",
```

should we ? not sure

---

well even I'm not sure had confusion on this

---

lets keep it it as it is increases readability